File: bot.php
<?php
require_once __DIR__ . '/vendor/autoload.php';

$config = json_decode(file_get_contents(__DIR__ . '/config.json'), true);

// create an instance of the bot
$bot = new \TelegramBot\Api\Client($config['token']);

// define an example command
function hello($message) {
  global $bot;
  $bot->sendMessage($message->getChat()->getId(), "Hello " . $message->getFrom()->getFirstName());
}

$bot->command('start', function($message) {
  hello($message);
});

$bot->run();

File: bot.rb
require 'telegram/bot'
require 'json'

# parse config.json
config = JSON.parse(File.read('config.json'))

# define an example command
def hello(message)
  Telegram::Bot::Client.run(config['token']) do |bot|
    bot.api.send_message(chat_id: message.chat.id, text: "Hello #{message.from.first_name}")
  end
end

# start the bot
Telegram::Bot::Client.run(config['token']) do |bot|
  bot.listen do |message|
    case message.text
    when '/start'
      hello(message)
    end
  end
end
